05_syntax_lab/03.py

```diff
@@ -1,7 +1,13 @@
-"""
-Write a program that randomizes a number
-and prints the sum total of its digits.
-For example if the number was: 2345
-The result should be: 14
-"""
-
+from random import randint
+"""
+Print the sum of a the digits of a random number between 1-10000
+"""
+
+
+input=str(randint(1,10000))
+print "Selected random number as string "  , input
+sum=0
+for oneChar in input:
+    sum += int(oneChar)
+
+print "The sum of the digits is : " ,sum

```

Comment:
string.format is generally preferred for printing: `print "The sum is {}".format(sum)`